### ISSUE TITLE
Update ember-cli-babel

### ISSUE THREAD

**Issue by steveszc** [adopted-ember-addons/ember-pikaday#175]
The ember-cli-babel dependency should be updated to the latest version to allow consuming apps to drop ember-cli-shims as a dependency of ember 2.16+ apps.
https://www.emberjs.com/blog/2017/11/20/ember-2-17-released.html#toc_changes-in-ember-cli-2-17

**Comment by duizendnegen** [comment]
Welcoming a PR which had been tested to work :)

**Comment by jrjohnson** [comment]
I think this is resolved, just needs a release.

**Comment by mdcoopir** [comment]
When will this be released?

**Comment by alexlafroscia** [comment]
Since it's unclear if/when `master` will be cut into a new release, I published a fork at `@alexlafroscia/ember-pikaday` from the current `master` branch (commit e367b56a242fd1cafb1940788bcac2308b31fa53)

**Comment by Frozenfire92** [comment]
note that deprecation warnings will appear when on a newer version of ember (3.3)

```
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version5.2.8 located: my-app -> ember-pikaday -> ember-cli-babel
DEPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version5.2.8 located: /my-app/node_modules/ember-pikaday/node_modules/ember-cli-babel
```

**Comment by jrjohnson** [comment]
This is going to become critical as ember-cli 3.4 seems to drop support for addons which have not updated in https://github.com/ember-cli/ember-cli/pull/7774#issuecomment-384677373

**Comment by jonnii** [comment]
@alexlafroscia thank you for publishing a new release, it's taking the pressure off.

**Comment by jrjohnson** [comment]
@alexlafroscia If you're up for maintaining this package take a look at https://www.npmjs.com/policies/disputes
NPM has a clear process for adopting an abandoned package and making your fork official.

**Comment by alexlafroscia** [comment]
Maybe we can do this more formally, if people are interested in that. I don't mind taking over maintenance duties, but maybe we can transfer ownership to https://github.com/adopted-ember-addons and do something on the NPM side to allow a group of people to publish updates from that repo.

@edgycircle would that be OK with you?

**Comment by jrjohnson** [comment]
@alexlafroscia can you volunteer over in #195

**Comment by alexlafroscia** [comment]
I'll join that conversation, thanks @jrjohnson!

**Comment by alexlafroscia** [comment]
Okay, seeing as I just released a _real_ version of `2.2.5` on the official `ember-pikaday` package, I'm going to close this.

If you've using my fork at `@alexlafroscia/ember-pikaday`, please move back to the original `ember-pikaday`. The `2.2.5` package is identical to my fork's `2.2.5`.
